src/MSALWrapper/PCAWrapper.cs

```diff
@@ -78,6 +91,7 @@ public async Task<TokenResult> GetTokenInteractiveAsync(IEnumerable<string> scop
                 {
                     Title = this.PromptHint,
                 })
+                .WithUseEmbeddedWebView(this.UseEmbeddedWebView)

```

Comment:
Above this we make a call to `.WithEmbeddedWebViewOptions()`. Is that still necessary if we've written this code to never use the embedded web view?

Reply:
In theory that is also how we are supposed to set the prompt hint for WAM, but I'm not sure that's actually the case. I could have sworn we saw WAM prompts using the prompt hint in testing. But now I'm not seeing WAM use the prompt hint. I think we can open a follow up bug to figure that out.